packages/ai-jsx/src/core/opentelemetry.ts

```diff
@@ -66,9 +78,6 @@ export function openTelemetryStreamRenderer(streamRenderer: StreamRenderer): Str
             const resultIsPartial = result.find(isElement);
             // Record the rendered value.
             span.setAttribute('ai.jsx.result', resultIsPartial ? debug(result, true) : result.join(''));
-            if (!resultIsPartial) {
-              span.setAttribute('ai.jsx.result.tokenCount', getEncoder().encode(result.join('')).length);
-            }
```

Comment:
lol I'm annoyed with myself for doing something that was basically useless... was there a good way for me to have known that this wasn't right?

Reply:
Meh, I wouldn't go so far to say it's useless:
- It's not obvious from this code that there's extensive usage of partial rendering for conversations
- Maybe one hint of trouble is that there's no way to use the correct tokenizer here? (If we did token counting for Anthropic/Llama, that is.)
- That this code is far away from anything else that does token counting is also a bit dicey